modules/proxy/proxy_util.c

```diff
@@ -403,8 +419,12 @@ PROXY_DECLARE(char *)
     return NULL;
 }
 
-PROXY_DECLARE(int) ap_proxyerror(request_rec *r, int statuscode, const char *message)
+static int ap_proxyerror_ex(request_rec *r, int statuscode, const char *message,
```

Comment:
Is this the correct name for a `static` function? I think it should be either `proxyerror_core` (at least dropping the `ap_` prefix, the further part of the name I don't care that much) or we should make it part of the API and properly declare it via `PROXY_DECLARE` as non static function and add it to `mod_proxy.h`.

Reply:
Done in [8ad4da6](https://github.com/apache/httpd/pull/367/commits/8ad4da692850d480bf2fd85518485596872e1b3b)